### PR TITLE
rename JAVAX_API to JAKARTA_API in ee9 and ee10 Source

### PR DESCRIPTION
**Pull request by lachlan-roberts**

### CODE REVIEW

File: jetty-ee10/jetty-ee10-annotations/src/test/java/org/eclipse/jetty/ee10/annotations/TestAnnotationIntrospector.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public void testIsIntrospectable() throws Exception
             assertTrue(introspector.isIntrospectable(new ServletE(), holder));
 
             //a JAVAX API sourced servlet can be introspected
```
Comment (joakime) [inline jetty-ee10/jetty-ee10-annotations/src/test/java/org/eclipse/jetty/ee10/annotations/TestAnnotationIntrospector.java]:
```suggestion
            //a JAKARTA API sourced servlet can be introspected
```

File: jetty-ee9/jetty-ee9-annotations/src/test/java/org/eclipse/jetty/ee9/annotations/TestAnnotationIntrospector.java [hunk 2]
```diff
@@ -52,7 +52,7 @@ public void testIsIntrospectable() throws Exception
             assertTrue(introspector.isIntrospectable(new ServletE(), holder));
 
             //a JAVAX API sourced servlet can be introspected
```
Comment (joakime) [inline jetty-ee9/jetty-ee9-annotations/src/test/java/org/eclipse/jetty/ee9/annotations/TestAnnotationIntrospector.java]:
```suggestion
            //a JAKARTA API sourced servlet can be introspected
```

**Review by joakime — CHANGES_REQUESTED** [review]
Some javadoc you missed.

Also, what about the JAVAX references in jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/util/ReflectUtils.java ?

**Comment by lachlan-roberts** [comment]
@joakime the `ReflectUtils` is used for both Javax and Jakarta websocket implementations. So I think it needs both.

**Review by olamy — APPROVED** [review]

**Review by joakime — APPROVED** [review]

**Pull request merged**